src/main/java/appeng/util/Platform.java

```diff
@@ -803,7 +805,20 @@ public static boolean isWrench( final EntityPlayer player, final ItemStack eq, f
 			{ // explodes without BC
 
 			}
-
+			
+			// CoFH Wrench Support
+            try {
+                if( eq.getItem() instanceof IToolHammer )
+                {
+                    IToolHammer wrench = (IToolHammer) eq.getItem();
+                    return wrench.isUsable( eq, player, pos );
+                }
+            }
+            catch( final Throwable ignore )
+            {
+            
+            }
+            
 			if( eq.getItem() instanceof IAEWrench )
 			{
 				final IAEWrench wrench = (IAEWrench) eq.getItem();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Method appeng.util.Platform.isWrench(EntityPlayer, ItemStack, BlockPos) appears to call the same method on the same object redundantly [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.orod.org/coding_rules#rule_key=fb-contrib%3APRMC_POSSIBLY_REDUNDANT_METHOD_CALLS)